mock/services.go

```diff
@@ -309,6 +312,11 @@ func (m *InvoicesService) MarkFailed(invoiceNumber int) (*recurly.Response, *rec
 	return m.OnMarkFailed(invoiceNumber)
 }
 
+func (m *InvoicesService) RefundVoidOpenAmount(invoiceNumber int, amountInCents int, refundApplyOrder string) (*Response, *Invoice, error) {
+	m.RefundVoidOpenAmountInvoked = true,
```

Comment:
Removed in [e9b88a](https://github.com/blacklightcms/recurly/pull/11/commits/1b52bfba87d06be765b03b13c108236146e9b88a). I actually think the problem was that `*Response` > `*recurly.Response`.